package.json

```diff
@@ -17,7 +17,7 @@
     ]
   },
   "dependencies": {
-    "@opam/dune": "1.11.3",
+    "@opam/dune": ">=1.6.0",
```

Comment:
To avoid pinnig problem in the future but in fact it coulb be lower  but ill formed dune depencie in @reason-native/console avoid it.